actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -298,25 +294,27 @@ def link_to(name = nil, options = nil, html_options = nil, &block)
       #   #      <input name="authenticity_token" type="hidden" value="10f2163b45388899ad4d5ae948988266befcb6c3d1b2451cf657a0c293d605a6"  autocomplete="off"/>
       #   #    </form>"
       #
-      #   <%= button_to "Create", { action: "create" }, remote: true, form: { "data-type" => "json" } %>
-      #   # => "<form method="post" action="/images/create" class="button_to" data-remote="true" data-type="json">
-      #   #      <button type="submit">Create</button>
-      #   #      <input name="authenticity_token" type="hidden" value="10f2163b45388899ad4d5ae948988266befcb6c3d1b2451cf657a0c293d605a6"  autocomplete="off"/>
-      #   #    </form>"
-      #
```

Comment:
I believe this example also works without `remote: true`, so perhaps we can just remove that part instead.